packages/cashier/src/components/cashier-locked/cashier-locked-provider.tsx

```diff
@@ -53,6 +57,21 @@ const getMessage = ({
     const pa_commision_withdrawal_limit = cashier_validation?.includes('PACommisionWithdrawalLimit');
     const pathname = history.location.pathname;
 
+    if (is_account_to_be_closed_by_residence) {
+        return {
+            icon: 'IcCashierNoBalance',
+            title: localize('Deposits disabled'),
+            description: (
+                <Localize
+                    i18n_default_text='Due to business changes, client accounts in Senegal are to be closed. Withdraw any remaining funds by {{date}}.'
```

Comment:
@wojciech-deriv thought about this and based on this created variables without country name, for example `is_account_to_be_closed_by_residence`, `trading-disabled-by-residence-modal` and next, but I don't know what to do with translation. As I know we can't use variables there (in Localize and localize) which we need to translate, we can use just values which will not be translated... In this case I've created all components, but in future we can just change `Senegal` to the needed country name. Or, maybe I'm wrong, and you know how to translate variable inside of Localize component?

Reply:
@wojciech-deriv regarding the content like "client accounts in your country", I can't change it because we have content team that is responsible for this. Also, this is high priority card because we need to close Senegal account in next 1-2 months and not sure somebody will approve content change because we need this code on production ASAP